src/Stripe.net/Services/Sources/SourceSourceOrderItemOptions.cs

```diff
@@ -0,0 +1,23 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class SourceSourceOrderItemOptions : INestedOptions
+    {
+        [JsonProperty("amount")]
```

Comment:
Those have official docs in the API reference. Could you add those?

Reply:
Added doc strings from the API reference.